normandy/recipes/models.py

```diff
@@ -68,14 +76,17 @@ class Recipe(models.Model):
     # Fields that determine who this recipe is sent to.
     enabled = models.BooleanField(default=False)
     filter_expression = models.TextField(blank=False)
-    approver = models.ForeignKey(User, null=True, blank=True)
+    approval = models.OneToOneField(Approval, related_name='recipe', null=True, blank=True)
+
+    # A tuple of fields that can be edited without causing the recipe to be disabled
+    EDITABLE_FIELDS_WHITELIST = ('name', 'enabled', 'approval',)
```

Comment:
Oh, I see! Enabled being in the whitelist means that disabling a recipe does not remove its approval! Clever!